docs/source/gbd2017_models/causes/covid_19/index.rst

```diff
@@ -11,35 +11,285 @@ Covid-19 Custom SEIR Vivarium Cause Model
+
 Scope
 +++++
 
-Assumptions and Limitations
-+++++++++++++++++++++++++++
+The immediate intention of this cause model will be to aid in modeling various scenarios relating to varied population density, mobility, and other potential covid-19 control measures at the University of Washington and immediately surrounding areas/populations. This covid-19 Vivarium cause model is intended to make use of the data and framework of the existing IHME covid-19 forecast model SEIR component (briefly described above). The scope of this cause model document is intended to develop a basic strategy for an infectious SEIR cause model and will start with minimal complexity and can be expanded upon if it becomes necessary.
+
+Notably, this is currently intended to be a **mortality-only** cause model and does not indend to measure years lived with disability (YLDs).
+
 
 Model Population/Demography
 +++++++++++++++++++++++++++
 
+.. todo::
+
+	Define demographic model
+
 Cause Model Diagram
 +++++++++++++++++++
 
+
+			:math:`S` --> :math:`E` --> :math:`I_1` --> :math:`I_2` --> :math:`R`
+
+.. list-table:: State Definitions
+   :widths: 5 25
+   :header-rows: 1
+
+   * - Parameter
+     - Definition
+   * - :math:`S`
+     - Susceptible
+   * - :math:`E`
+     - Exposed, but not infectious
+   * - :math:`I_1`
+     - Infectious (define first stage)
+   * - :math:`I_2`
+     - Infectious (define second stage)
+   * - :math:`R`
+     - Recovered
+
 State and Transition Data Tables
 ++++++++++++++++++++++++++++++++
 
 State Data
 ^^^^^^^^^^
 
+Prevalence
+""""""""""
+
+Simulants will be initialized into covid-19 disease model states based on the forecasted IHME covid-19 model estimates. 
+
+.. todo::
+
+	Detail specific data source and values
+
+Mortality
+"""""""""
+
+We will model covid-19 mortality using an age-dependent infection fatality ratio (IFR), as consistent with the IHME forecast model. There will be two steps to this piece of the model, the first being the determination of if an individual dies due to covid-19 (described here), and the being determining *when* they exit their current model state (decribed in the `Transition Data`_ section).
+
+In the same time-step for which a simulant is initialized into the :math:`E`, :math:`I_1`, or :math:`I_2` covid-19 model states or transitions into the :math:`E` model state, the age-specific IFR should be used to determine if that simulant will die of covid-19, such that the IFR shown in the table below represents the probability that an infected individual dies from covid-19. 
+
+.. csv-table:: Infection Fatality Ratios
```

Comment:
Chris indicated that the higher burden of COVID among BIPOC is due to a higher proportion of BIPOC being essential workers, rather than, say, lower quality or lack of access to healthcare---but it would be interesting to see if there's any data out there on IFR stratified by race.